### ISSUE TITLE
7 days market restriction

### ISSUE THREAD

**Issue by Daeroz** [Maxx53/SCMBot#283]
Should i wait my browser's restriction before putting cookies into SCM settings?
Is there any difference between botting with cookies/working browser's market and without cookies/restricted browser's market?

Just tested without cookies/with restricted bser market on cheap skins. I've used both tabs (recent and item), bot delay 333, item delay 0 for both lists, 6 hosts with 50~ ping, others are default.

I was buying 0.1$ skin for 0,05$ from myself. After i set this skin it took about 15 seconds before it was purchased by someone else. There were no info about this item in my bot (nothing in recent tab and no price changes in item tab). I tried this several times with the same result.

Am i doing something wrong?

###############################################################

Нужно ли ждать активации маркета у меня в браузере?
Есть ли разница между запуском бота вместе с куки и рабочим маркетом в браузере и просто обычным запуском с рабочим маркетом на стиме?

Пробовал без вставки куки и с закрытым маркетом в хроме на дешевых скинах. Скин ставил в обе вкладки (недавние и обычную), задержка 333, задержка на вещь 0, 6 хостов с пингом 50~
, все остальное дефолт. 

Пробовал купить у себя 3-х рублевый скин за 1,5. Выставил предмет и через 15 секунд он уже кому-то ушел. Никакой инфы о нем в боте не прошло (ни в  недавних поступлениях, даже цена не показалась в первой вкладке). Пробовал с несколькими скинами, результат тот же.

Может я что-то делаю не так?

**Comment by Maxx53** [comment]
Боту нужны куки, держать открытым параллельно браузер или клиент совсем не обязательно. Куки можно импортировать из браузера или залогиниться в самом боте с активацией стимгарда и последующим ожиданием в 7 дней.
Я правильно понял, ты пытаешься выставить предмет с одного аккаунта и увидеть его в боте на другом аккаунте. В этом случае есть вероятность успеха, если предмет не особо популярный (твой предмет будет держаться сверху хоть пару секунд).

**Comment by Daeroz** [comment]
Да, меня просто удивило что из 5 попыток ни разу не написало о моем предмете, хотя держались они около 10-15 секунд. Сейчас проверил в настройках поля куки пустые, хотя вводил их точно, есть шанс что они слетели после перезагрузки?
Как я понял не важно что у меня в браузере 7 дней висит, на работу это влиять не должно?
И где можно взять новые хосты, на случай если старые откажут? сайт который ты давал не дает результатов http://startping.com/ping/steamcommunity.com/5430f3128c3dcff231dffb23

Заранее спасибо.

**Comment by Maxx53** [comment]
Поля в окне импорта куки предназначены только для записи значений в контейнер, который после закрытия программы сохраняется в файл. Контроль успеха импорта осуществляется через Логин, а затем покупку/продажу любого предмета на площадке.
Если куки импортировались из браузера, в котором действует ограничение в 7 дней, соответственно, бот с площадкой работать не сможет.
Хосты ищем следующим образом: забиваем в любой поисковой системе запрос типа "пинг из другой страны", найдутся ресурсы аналогичные startping.com, их немало.

**Comment by Daeroz** [comment]
Спасибо, теперь все понятно.
Последний вопрос: существует ли безопасная задержка для перепродажи вещей?
